src/pip/_internal/resolution/resolvelib/candidates.py

```diff
@@ -102,6 +102,9 @@ def make_install_req_from_dist(dist, parent):
 
 
 class _InstallRequirementBackedCandidate(Candidate):
+    # These are not installed
+    is_installed = False
```

Comment:
Heh, I didn’t know Mypy accepts this, TIL. (Always assumed you need to override a property with a property.)

Reply:
lol, honestly I just got fed up of everything needing to be a property and decided I was going to make an attribute and mypy be damned 🙂 I was pleasantly surprised when it worked.